src/ORM/Connect/DBSchemaManager.php

```diff
@@ -28,6 +30,16 @@ abstract class DBSchemaManager
      */
     private static $check_and_repair_on_build = true;
 
+    /**
+     * For large databases you can declare a list of DataObject classes which will be excluded from
+     * CHECK TABLE and REPAIR TABLE queries during dev/build. Note that the entire inheritance chain
+     * for that class will be excluded.
```

Comment:
```suggestion
     * for that class will be excluded, including both ancestors and descendants.
```

Reply:
I explained this in more detail here https://github.com/silverstripe/developer-docs/pull/201#issuecomment-1493725956

Reply:
That's what "the entire inheritance chain for that class" means, but there's no harm adding this.
Done.